src/styles/blocks/header.scss

```diff
@@ -0,0 +1,11 @@
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.logo {
+  height: 40px;
+}
```

Comment:
Move it to a separate file or make it an element

Reply:
i think u can make it, element of header block  or move it to a separate file